requirements-dev.txt

```diff
@@ -0,0 +1,9 @@
+# Python dependencies for development
```

Comment:
why not just call this file requirements.txt? what makes it dev?

Reply:
If this is just requirements.txt, then I prefer not to list cleanvision at all, because it gets too complicated to track between developer-branch version and latest stable release version. Definitely shouldn't pin the version as done below IMO

Reply:
I pinned the version because sometimes pip installs a stale version that might be in its cache, this forces it to install the specific version.

Reply:
I kept it dev, because cleanvision is explicitly installed inside notebooks and one could just download the repo and run them if they already have Jupyter.

Reply:
Removed cleanvision from requirements, didn;t want to track it through releases.